build/credscan-exclusion.json

```diff
@@ -25,10 +29,6 @@
      "file": "XForms.UWP_TemporaryKey.pfx", 
       "_justification": "Legitimate UWP app certificate file with private key required for the app to function. this is sample test app."
     },
```

Comment:
could potentially have used the same name for the app... then the addition would not have been needed.